Startup.cs

```diff
@@ -20,6 +22,14 @@ public Startup(IConfiguration configuration)
       // This method gets called by the runtime. Use this method to add services to the container.
       public void ConfigureServices(IServiceCollection services)
       {
+         var connection = @"Server=db;Database=Singer;User=sa;Password=singer-2019-password;";
```

Comment:
Shouldn't this be taken from `appsettings.Development.json`?

Reply:
Agreed, will fix.

Reply:
However, there's already a `*.mdf` in the `.gitignore`